README.md

```diff
@@ -1,2 +1,172 @@
 # db-muffler
-Tool to obscure values in your SQL dump file
+
+> Tool to obscure values in your SQL dump file
+
+## Idea
+
+This tool will allow to obscure specific fileds within INSERT statements of your SQL dump file.
+Could be a good tool to hide sensitive data or just replace your dev data with more realistic one.
+
+## Installation
+
+Global installation _(suggested)_
+
+```
+$ npm i -g db-muffler
+$ db-muffler
+```
+
+Local installation
+
+```
+$ npm install db-muffler
+$ node_modules/.bin/db-muffler
+```
+
+## Usage
+
+```
+Usage
+    $ db-muffler <file_path>
+
+Options
+    --test, -t    Test run which would not overwrite input file
+    --output, -o  Output file where the result will be store. Default value is the input file name
+    --config, -c  Config file where developer can store rules on how to obscure DB data. Default value is ./db-muffler.json
+
+Examples
+    $ db-muffler ./inputfile.sql
+    Will use ./db-muffler.json config file and then will run muffler to obscure
+    data and then output the result in inputfile.sql
```

Comment:
So this overwrites inputfile.sql with the output?

Reply:
In this example yes. Not fully sold on the idea but I guess once you got the SQL dump the default operation is to overwrite the file. What do you think about this @ansonkao really interested in your feedback

Reply:
Fair enough, I would just make it explicit in the README that default is overwrite, wasn't sure if it was a typo. Will give it a spin next week!

Reply:
Actually I have a description above `Output file where the result will be store. Default value is the input file name` @ansonkao what do you think would be a better documentation to explain this behaviour?

Reply:
Suggested changes below:
```
Options
    --test, -t    Test run which would not overwrite input file
    --output, -o  Output file where the result will be **stored. By default, it will overwrite the input file.**
    --config, -c  Config file where developer can store rules on how to obscure DB data. Default value is ./db-muffler.json

Examples
    $ db-muffler ./inputfile.sql
    Will use ./db-muffler.json config file and then will run muffler to obscure
    data and **then overwrite inputfile.sql with the result**
```